.gitignore

```diff
@@ -24,6 +24,7 @@ build/
 bld/
 [Bb]in/
 [Oo]bj/
+localpkg/
```

Comment:
This is fixing annoying existing problem: full CoreCLR build will create a bunch of these localpkg directories that will show up in `git status`:

```
D:\coreclr>git status
On branch master
Your branch is up-to-date with 'origin/master'.
Untracked files:
  (use "git add <file>..." to include in what will be committed)

        src/.nuget/Microsoft.NETCore.ILAsm/localpkg/
        src/.nuget/Microsoft.NETCore.ILAsm/win/localpkg/
        src/.nuget/Microsoft.NETCore.ILDAsm/localpkg/
        src/.nuget/Microsoft.NETCore.ILDAsm/win/localpkg/
... bunch more of these ...
```

@weshaggard Is there more appropriate way to fix it?